### ISSUE TITLE
The opened windows on Windows does not have the right height

### ISSUE THREAD

**Issue by jbdamiano** [p-sam/electron-prompt#44]
On Windows the displayed prompt is too small. On OS X the displayed prompt is correct.

![image](https://user-images.githubusercontent.com/3286076/65310171-fd6a8900-db8d-11e9-997a-55bba05c763c.png)

**Comment by p-sam** [comment]
Could you show the code used to invoke the prompt window?

**Comment by jbdamiano** [comment]
```
const ePrompt = require('electron-prompt');

ePrompt({
     title: 'Enter the Terminal Profile Data',
     label: 'data:',
     value: ''
})
 .then((r) => {
     if(r !== null) {
        //Send a message to decode the parameter
        //win.openDevTools();
        win.webContents.send('decode', {
            data: r`
         });
     }
})
.catch(console.error);
```

**Comment by Fraasi** [comment]
I remember this happening to me too. If you don't set the height, it defaults to 130, which is not enough even for this kind of simple input window on windows. My guess is that on OS X the font/input box/buttons/padding or whatnot is different size than on windows and everything fits snuggly.

Probably only way around this is to check which platform you are on and set height accordingly.
Maybe it wouldn't be too difficult to change the fixed default height to fit to content instead?

**Comment by p-sam** [comment]
I'd probably prefer a solution that would just set a window client size instead of trying to find the correct window size (including borders) on each platform.

**Comment by Fraasi** [comment]
Yeah, that was my tired attempt to explain on the last line 😩 To set the window height automatically according to the content the default behaviour. I'll try to find time to play around with this in the next few days and see if it's easy to implement.

**Comment by Fraasi** [comment]
Ok, it seems there's something weird going on with my win10 machine.  
Running the latest electron quick start and electron-prompt 1.3.1.
If you leave the height out, prompt should default to 130, but it's around 80 pixels...  

![pic2](https://user-images.githubusercontent.com/22402899/66080759-6af3bd80-e56f-11e9-803e-1688b9071b8f.PNG)

If you put the height to 330, it's only around 280...

![pic1](https://user-images.githubusercontent.com/22402899/66080777-72b36200-e56f-11e9-9906-2ee1b07efb4b.PNG)

So where's the missing 50 pixels:thinking:?

As a side note, I wrote [this little function](https://github.com/Fraasi/Saeae/blob/master/src/assets/resizeWindow.js) about a year ago for one of my apps to resize the electron window height to fit all the content in the window without scroll bars. Might be useful when thinking a solution for this issue.

**Comment by Fraasi** [comment]
Found the missing pixels, culprit is the BrowserWindow option [useContentSize](https://electronjs.org/docs/api/browser-window#new-browserwindowoptions) , if set to false, I get the default 130 pixels for the height, but as you can see, it's not enough...

![pic3](https://user-images.githubusercontent.com/22402899/66106560-d6b54500-e5c6-11e9-8628-bb302e086509.PNG)

Either there's some weird conflict with the settings or this is a bug in electron?
Yea, related electron [issue](https://github.com/electron/electron/issues/14308)

**Comment by Fraasi** [comment]
Ok, after reading the electron issue and testing, simply setting the 'resizable' to true solves this issue

![pic4](https://user-images.githubusercontent.com/22402899/66109569-a329e900-e5cd-11e9-9611-c9c1a3654c3c.PNG)

So, I would propose that we change the resizable-option default to true, @p-sam ?

**Comment by p-sam** [comment]
Yeah, but as the option states, that would also make the window resizable, which is not desirable. You seem to want to set the width and height dynamically based on the content, and as I stated in my previous message, the ideal solution would be to set something commonly called as the "client size", which is the inner size of the window content (without chrome/border). I may look into this myself later, however I have other stuff on my TODO right now, sorry

**Comment by Fraasi** [comment]
No worries, I was just curious about what caused this behaviour. Anyway, it's should be pretty easy for others to 'fix' this now if they bumb into this problem.

**Comment by p-sam** [comment]
Please check if everything is resolved on develop. The actual issue was that the minimum height changed for all platforms (and was wrong), however not all platforms would exhibit the same behavior with different values of resizable/useContentSize. I'll probably release a new version I have multiple confirmations that it works now. (Tested myself on Windows and OSX right now)

**Comment by Fraasi** [comment]
Ok, glad to hear you found the actual problem. Comparing the master and developement branches, I see there is few new options and a lot of housecleaning done. Looks good to me for the next release 🚀

**Comment by p-sam** [comment]
alright, let's go then

**Comment by p-sam** [comment]
fixed in v1.4.0
